docs/administration/configure-proxy-cache/_index.md

```diff
@@ -26,6 +26,8 @@ The next time a user requests that image, Harbor checks the image's latest manif
 * If the target registry is not reachable, the proxy cache project serves the cached image.
 * If the image is no longer in the target registry, no image is served.
 
+As of Harbor v2.1.1, when checking an image on Docker Hub, none of the blobs for the image are updated. This means that when Harbor performs these checks on the image, it will not count towards Docker Hub's [rate limit policy](https://www.docker.com/blog/scaling-docker-to-serve-millions-more-developers-network-egress/).
```

Comment:
This is not entirely accurate.  Here's how I would phrase it

"As of Nov 1, 2020, Dockerhub’s new policy on [download rate limiting](https://www.docker.com/blog/scaling-docker-to-serve-millions-more-developers-network-egress/). goes into effect.  This was announced as part of its Docker Terms of Service (ToS) communications.  The specific pull rates are as follows:

Free plan – anonymous users: 100 pulls per 6 hours 
Free plan – authenticated users: 200 pulls per 6 hours
Pro plan – unlimited
Team plan – unlimited

A pull is defined as a manifest GET request to Docker Hub.  This means that even if all the layers that make up an image already exist locally, running a docker pull command will still register as a pull towards the rate limit.

Luckily, the Harbor Proxy Cache will not trigger the Docker Hub rate limiter if no image layers have been updated since it's only firing a HEAD request for determining whether the cached image has been updated.